README.rst

```diff
@@ -64,38 +64,38 @@ Assuming EUPS is available on your system, simply::
 By Modifying your Environment
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-After cloning, add the ``alert_packet/python`` directory to your ``PYTHONPATH`` environment variable, and the ``alert_packet/bin`` directory to your ``PATH`` environment variable.
+After cloning, add the ``alert_packet/python`` directory to your ``PYTHONPATH`` environment variable, and the ``alert_packet/python/lsst/alert/packet/bin`` directory to your ``PATH`` environment variable.
```

Comment:
Are we really telling people to add that directory to their PATH? Is this because you don't want people to install it?

Reply:
I inherited this repo, and don't really understand this section myself; I'm erring on the side of changing one thing at a time. I agree with you, I think. I suspect that this was in the README just because it was expedient, but I don't really know.

Reply:
I think you should assume that all pre-existing installation instructions were based on expediency: there was no meaningful installation system until you added one. :-) I'd suggest removing anything that isn't obviously correct.

Reply:
Sorry [Chesterton](https://en.wikipedia.org/wiki/Wikipedia:Chesterton%27s_fence), we're bulldozing all the fences today!

(sounds good to me :))